website/docs/guides/dependencies.md

```diff
@@ -58,6 +58,19 @@ fn main() -> felt252 {
 }
 ```
 
+## Development dependencies
+
+You can add a `[dev-dependencies]` section to your Scarb.toml whose format is equivalent to `[dependencies]`:
+
+```toml
+[dev-dependencies]
+tempdir = "0.3"
```

Comment:
Please use real cairo packages as examples. In this case it could be alexandria.